portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.data;
+
+/** An item on a todo list. */
+public final class Comment {
+
+  private final long id;
+  private final String timestamp_UTC;
+  private final String name;
+  private final String email;
+  private final String subject;
+  private final String comments;
+
+  public Comment(long id, String timestamp_UTC, String name, String email, String subject, String comments) {
+    this.id = id;
+    this.timestamp_UTC = timestamp_UTC;
+    this.name = name;
+    this.email = email;
+    this.subject = subject;
+    this.comments = comments;
+  }
+}
```

Comment:
nit: could you please leave an empty line at the end of the file.

https://google.github.io/styleguide/javaguide.html#s4.3-one-statement-per-line